homeassistant/components/amcrest/camera.py

```diff
@@ -309,6 +310,11 @@ def model(self):
         """Return the camera model."""
         return self._model
 
+    @property
+    def unique_id(self) -> str:
+        """Return a unique id for this instance."""
+        return self._serial
```

Comment:
```suggestion
        return self._serial if self._serial != "unknown" else None
```
The `unique_id` should never be "unknown"

Reply:
Thanks - I started with two properties, and then simplified it to one, but with this I made the error of allowing serial to potentially be unknown.